lib/graphql/fragment_cache.rb

```diff
@@ -81,6 +82,7 @@ def verify_interpreter_and_analysis!(schema_defn)
     end
 
     self.cache_store = MemoryStore.new
+    self.default_options = {}
```

Comment:
Just an idea: do we want to make sure that user does not pass something wrong to the hash? (e.g., `sHema_cache_key`)

Reply:
I am personally indifferent to it! How would you check for something wrong in the hash though?

For example we could go over each key manually against a predefined set of allowed keys. Knowing that we may not be able to validate everything, since there may be options specific to the cache store. Although I can't think of any other option than `expires_in` in practice, so validating for a set of allowed keys + expires_in may be good enough for most cases.

Or we could just have discrete options rather than a separate `default_options` hash. May be cleaner, although we'd have to manually "merge" for each option in the code. For example:

```ruby
GraphQL::FragmentCache.configure do |config|
  config.expires_in = 1.hour
  config.schema_cache_key = nil
  config.cache_key = :object
end
```

Reply:
> as currently the codebase doesn't validate when someone passes a wrong option 

True, let's keep the current approach for now and see if someone complains 🙂